src/pages/index.js

```diff
@@ -1,99 +1,194 @@
 import { initialCards, formSettings } from "../utils/constants.js";
 import "../pages/index.css";
-/* -------------------------------- elements -------------------------------- */
 
+/* ------------------------------ API Instance ----------------------------- */
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "d7e04e76-ddf3-44aa-931a-ba12647359c4",
+    "Content-Type": "application/json",
+  },
+});
+
+/* ------------------------------ User Info ----------------------------- */
+const userInfo = new UserInfo({
+  name: ".profile__title",
+  description: "#profile__description",
+  avatar: ".profile__image",
+});
+
+/* ------------------------------ DOM Elements ----------------------------- */
 const profileEditButton = document.querySelector("#profile-edit-button");
 const profileEditModal = document.querySelector("#profile-edit-modal");
-//const profileCloseButton = profileEditModal.querySelector("#modal-close-button");
-const profileTitle = document.querySelector(".profile__title");
-const profileDescription = document.querySelector("#profile__description");
 const profileTitleInput = document.querySelector("#profile-title-input");
 const profileDescriptionInput = document.querySelector(
   "#profile-description-input"
 );
 const profileAddButton = document.querySelector(".profile__add-button");
+const profileAvatarButton = document.querySelector(".profile__avatar-button");
 const profileEditForm = profileEditModal.querySelector(".modal__form");
-//const profileEditSubmit = profileEditModal.querySelector("#edit-save-button");
-//const cardListEl = document.querySelector(".cards__list");
-//const cardTemplate = document.querySelector("#card-template").content.firstElementChild;
-//const addModal = document.querySelector("#profile__add-form");
 const addForm = document.querySelector("#add__card-form");
-//const placeInput = document.querySelector("#addCard-title-input");
-//const placeInputUrl = document.querySelector("#addCard-description-input");
-//const closeButtonPlace = document.querySelector("#profile__close-modal");
-//const cardImageModal = document.querySelector("#card__image_modal");
-//const modalImageCloseButton = document.querySelector( "#modal__image-close-button");
-//const modalImageSrc = document.querySelector("#modal__image");
-//const modalImageDescription = document.querySelector("#modal__image_description");
-//const modal = document.querySelector(".modal");
-//const modals = document.querySelectorAll(".modal");
-//const modalContainer = document.querySelector(".modal__container");
-//const editModalWindow = document.querySelector("#profile-edit-modal");
-//const addModalWindow = document.querySelector("#profile-edit-button");
-
-// Define the necessary imports and elements as before
 
+/* ------------------------------ Form Validators ----------------------------- */
 const profileFormValidator = new FormValidation(formSettings, profileEditForm);
```

Comment:
**Could be improved**

You can **universally** create instances of validators for all forms in the project storing them inside one object `formValidators`.  And then you can take any validator using attribute `name` of the form where you need to disable/enable the submit button or remove errors.

```js
// define an object for storing validators
const formValidators = {}

const enableValidation = (config) => {
  const formList = Array.from(document.querySelectorAll(config.formSelector))
  formList.forEach((formElement) => {
    const validator = new FormValidator(config, formElement)
    // here you get the name of the form (if you don’t have it then you need to add it into each form in `index.html` first)
    const formName = formElement.getAttribute('name')

   // here you store the validator using the `name` of the form
    formValidators[formName] = validator;
    validator.enableValidation();
  });
};

enableValidation(config);

```

And now you can use them for disabling buttons or clearing errors:

```js

formValidators[ profileForm.getAttribute('name') ].resetValidation()

// or you can use a string – the name of the form (you know it from `index.html`)

formValidators['profile-form'].resetValidation()

```